.vsts-dotnet-ci.yml

```diff
@@ -65,7 +65,8 @@ jobs:
     name: Windows_x64_NetCoreApp21
     buildScript: build.cmd
     pool:
-      name: Hosted VS2017
+      name: NetCorePublic-Pool
+      queue: buildpool.windows.10.amd64.vs2017.open
```

Comment:
Should we instead build on VS2019?

Reply:
It probably doesn't matter either way. The only thing we use the VS on the machine for is the C++ compiler.

But if we wanted to update all of them to vs2019, that would be OK with me. But I don't think it is 100% required.